### PR TITLE
Topic/privx integration

### PR DESCRIPTION
**Pull request by jjh-ssh**
Compatible with PrivX version 26 onwards.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This is the first release of PrivX integration for PrivX 26 onwards.
Depends on demisto/privx-api:26.0.0.97238 dockerfile (so version number marked as 1.26.0 to indicate that).
Base container image will be upgraded later to v34, after that this PR will be finished.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrivX/Integrations/PrivX/PrivX.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+- defaultvalue: 443
+  display: Your PrivX server HTTPS port
+  name: port
+  required: true
+  type: 0
+- defaultvalue: privx-external
+  display: OAuth client ID
+  name: oauth-client-id
+  required: true
+  type: 0
```
Comment (Shellyber) [inline Packs/PrivX/Integrations/PrivX/PrivX.yml:20]: All the authentication params need to be of type 9 - [credentials](https://xsoar.pan.dev/docs/integrations/code-conventions#credentials)

**Review by Shellyber — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Great work.
Please see my comments and let me know if you need any help with it.
Also, let's schedule a demo for this new pack

**Comment by kgal-pan** [comment]
@jjh-ssh - Please sign the [Contribution Form](https://forms.gle/XDfxU4E61ZwEESSMA). Use `ssh-communications-security-17060` in the Partner ID field.

**Comment by Shellyber** [comment]
Hi @jjh-ssh,
Thank you for your contribution. 
I am closing  your PR for the moment.
Please feel free to re-open it when you are available to continue. 
Thanks again.

**Pull request closed without merging**